src/kem/saber/kem_saber_firesaber.c

```diff
@@ -51,6 +57,16 @@ OQS_API OQS_STATUS OQS_KEM_saber_firesaber_keypair(uint8_t *public_key, uint8_t
 		return (OQS_STATUS) PQCLEAN_FIRESABER_CLEAN_crypto_kem_keypair(public_key, secret_key);
 	}
 #endif /* OQS_DIST_BUILD */
+#elif defined(OQS_ENABLE_KEM_saber_firesaber_aarch64)
+#if defined(OQS_DIST_BUILD)
+	if (OQS_CPU_has_extension(OQS_CPU_EXT_ASIMD)) {
```

Comment:
I thought you eliminated ASIMD from PQClean META.yml?

Reply:
I did for the kyber meta.ymls but not sabre's. I'll end up reverting the sabre code so this will disappear.